packages/data/package.json

```diff
@@ -25,6 +25,7 @@
 		"@wordpress/compose": "file:../compose",
 		"@wordpress/element": "file:../element",
 		"@wordpress/is-shallow-equal": "file:../is-shallow-equal",
+		"@wordpress/priority-queue": "file:../priority-queue",
```

Comment:
It still needs to be added on PHP side.

Reply:
I didn't want to do it to avoid freezing its API as it becomes an official WP API. For now, I'm fine for this being duplicated in all the places it will be used as it's fairly small.

Reply:
It is marked as external since it’s listed in dependencies of the root package.json. Just saying to make it clear.